system/wymeditor/plugins/image_upload/jquery.wymeditor.image_upload.js

```diff
@@ -40,9 +40,14 @@ WYMeditor.editor.prototype.image_upload = function() {
 			},
 			complete: function(response){
 				response = response[0];
-				jQuery(".wym_src", doc).val(response.thumbUrl);
-				jQuery(".wym_alt", doc).val(response.original_filename);
-				jQuery("form#image_upload_form .submit", doc).val(oldSubmitLabel);
+				if(response.error){
+                    alert(response.error.substring(1));
+                    document.getElementById('upload-image').value = oldSubmitLabel;
```

Comment:
On error, you now do

    document.getElementById('upload-image').value = oldSubmitLabel;

and on success

                    jQuery("form#image_upload_form .submit", doc).val(oldSubmitLabel);

Those look like they might be doing the same, so maybe this can be one line outside of the if? Of is there a difference?

Reply:
Ah, missed that one, dragging stuff around. Thanks!